client/disputes/index.tsx

```diff
@@ -130,6 +131,15 @@ const getHeaders = ( sortColumn?: string ): DisputesTableHeader[] => [
 		isLeftAligned: true,
 		isSortable: true,
 	},
+	{
+		key: 'action',
+		label: __( 'Action', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Action', 'woocommerce-payments' ),
+		isLeftAligned: false,
+		isNumeric: true,
```

Comment:
To align this column right, only `isNumeric` needs to be set but this is counter-intuitive. Adding in `isLeftAligned: false` to make the intention clear.

https://github.com/woocommerce/woocommerce/blob/96c686c4d987469e95e3681c10109ffd1a322155/packages/js/components/src/table/README.md?plain=1#L252-L253